services/movie_session.py

```diff
@@ -0,0 +1,65 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: str
+) -> MovieSession:
+    if cinema_hall_id:
+        try:
+            cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+        except CinemaHall.DoesNotExist:
+            raise ValueError(f"CinemaHall with"
+                             f" id={cinema_hall_id} does not exist.")
+    if movie_id:
+        try:
+            movie = Movie.objects.get(id=movie_id)
+        except Movie.DoesNotExist:
+            raise ValueError(f"CinemaHall with id={movie_id} does not exist.")

```

Comment:
The error message should refer to 'Movie' instead of 'CinemaHall'. Correct it to 'Movie with id={movie_id} does not exist.'